CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## 0.0.3
```

Comment:
I think we don't need 0.0.1-2-3, as this is the very first PR to master. I would collapse everything in `[unreleased]` or just `0.0.1` . Also please add some disclaimer saying that this is a first prototype version that is not intended for production yet (both in CHANGELOG and README).